dockstore-webservice/src/main/resources/migrations.1.12.0.xml

```diff
@@ -102,4 +102,83 @@
             create unique index collection_displayname_index on collection (LOWER(displayname), organizationid);
         </sql>
     </changeSet>
+    <changeSet author="ktran (generated)" id="addMultipleOrcidPutCodesForVersionMetadata">
+        <createTable tableName="version_metadata_orcidputcode">
+            <column name="version_metadata_id" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="version_metadata_orcidputcode_pkey"/>
+            </column>
+            <column name="dbcreatedate" type="TIMESTAMP WITHOUT TIME ZONE"/>
+            <column name="dbupdatedate" type="TIMESTAMP WITHOUT TIME ZONE"/>
+            <column name="orcidputcode" type="TEXT"/>
+            <column name="userid" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="version_metadata_orcidputcode_pkey"/>
+            </column>
+        </createTable>
+        <addForeignKeyConstraint baseColumnNames="version_metadata_id" baseTableName="version_metadata_orcidputcode" constraintName="fk_version_metadata_id_version_metadata_orcidputcode" deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="id" referencedTableName="version_metadata"/>
+        <addUniqueConstraint columnNames="version_metadata_id, userid, orcidputcode" constraintName="unique_version_metadata_user_orcidputcode" tableName="version_metadata_orcidputcode"/>
+    </changeSet>
+    <changeSet author="ktran (generated)" id="addMultipleOrcidPutCodesForEntry">
+        <createTable tableName="entry_orcidputcode">
+            <column name="entry_id" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="entry_orcidputcode_pkey"/>
+            </column>
+            <column name="dbcreatedate" type="TIMESTAMP WITHOUT TIME ZONE"/>
+            <column name="dbupdatedate" type="TIMESTAMP WITHOUT TIME ZONE"/>
+            <column name="orcidputcode" type="TEXT"/>
+            <column name="userid" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="entry_orcidputcode_pkey"/>
+            </column>
+        </createTable>
+        <addUniqueConstraint columnNames="entry_id, userid, orcidputcode" constraintName="unique_entry_user_orcidputcode" tableName="entry_orcidputcode"/>
+    </changeSet>
+    <changeSet author="ktran" id="migrateOldOrcidPutCode">
```

Comment:
Could you add a note to the https://github.com/dockstore/dockstore-deploy/wiki/1.12-Release-Checklist to note that the release lead and probably at least one other person should have some exported workflows to test with ahead of the migration/release?

This is well tested but still would be nice to have a backup